src/test/java/be/seeseemelk/mockbukkit/inventory/InventoryMockTest.java

```diff
@@ -595,4 +595,15 @@ void closeInventoryRemoveViewer()
 		assertFalse(inventory.getViewers().contains(player));
 	}
 
+	@Test
+	void testClose()
+	{
+		Player player = server.addPlayer();
+		player.openInventory(inventory);
+		assertEquals(1, inventory.getViewers().size());
+		assertEquals(1, inventory.close());
+		assertEquals(0, inventory.getViewers().size());
+	}
```

Comment:
There should be an assertion to ensure the player no longer has the inventory open